addons/sale_project/controllers/portal.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import _lt
+from odoo import _
```

Comment:
Why did you change the function used for the translation? 🧐

Reply:
I replied on the enterprise PR